extensions/yarn/README.md

```diff
@@ -42,6 +46,39 @@ In cases where Phylum still needs to process some of the packages, the command
 will exit with a warning **without** installing the packages. Once the analysis
 is complete, another attempt can be made.
 
-[phylum]: https://phylum.io
-[phylum-cli]: https://github.com/phylum-dev/cli
-[yarn]: https://yarnpkg.com/
+## Troubleshooting
+
+There are some common reasons this extension may fail.
+
+### Sandbox violations
+
+The underlying package manager may fail due to operating within the Phylum
+sandbox. These errors are usually displayed in the output and say something
+about not being able to access a particular file or network resource.
+
+One possibility is that this is a valid sandbox violation that has not been
+accounted for with an exception. If you believe this is the case, please
+[submit an issue][issue] or [contact us][contact] to discuss adding exceptions.
+
+Another possibility is that a malicious action was taken by one of the packages.
+Blocking such malicious actions is a primary goal for using this extension but
+it may be hard to know what is malicious and what is not. If you need help
+making that determination, please [contact us][contact] and provide the log
+output along with any other relevant details.
```

Comment:
This is effectively just the same as the last paragraph. Why do we need to reiterate it again here?

Reply:
The "contact us" part? I will work to consolidate that.

Reply:
Yeah, I think once is fine for that.